packages/anoncreds/src/services/AnonCredsIssuerServiceOptions.ts

```diff
@@ -0,0 +1,44 @@
+import type {
+  AnonCredsCredential,
+  AnonCredsCredentialOffer,
+  AnonCredsCredentialRequest,
+  CredValue,
+} from '../models/exchange'
+import type { AnonCredsSchema } from '../models/vdr'
+
+export interface CreateSchemaOptions {
+  issuerId: string
+  name: string
+  version: string
+  attrNames: string[]
+}
+
+export interface CreateCredentialDefinitionOptions {
+  issuerId: string
+  tag: string // TODO: this was initially defined as optional, is that the case?
+  supportRevocation?: boolean
+
+  // If schema doesn't include the id, we need to add it as a separate input parameter
+  schema: {
+    value: AnonCredsSchema
+    id: string
+  }
+}
+
+export interface CreateCredentialOfferOptions {
+  credentialDefinitionId: string
```

Comment:
For all other methods it seems we pass the whole object (credential definition). The issue will be in that the createOffer needs the private part of the credential definition (which we can't get out of the indy-sdk).

Reply:
Only use IDs where absolutely necessary